stable/sematext-docker-agent/values.yaml

```diff
@@ -0,0 +1,23 @@
+image:
+  repository: sematext/sematext-agent-docker
+  tag: latest
+  pullPolicy: Always
+
+sematext:
+  # You will need to Create and Set at least one token before starting this agent!
+
+  # spmToken:
+  # logseneToken:
+
+  # Depending on where you created your apps, choose US or EU region.
+  region: US
+
+  useHostNetwork: true
+  name: sematext-docker-agent
+  resources:
+    requests:
+      cpu: 500m
+      memory: 128Mi
+    limits:
+      cpu: 1000m
+      memory: 1024Mi
```

Comment:
Is there a reason for these resource levels? Typically, if you look at other charts, these are commented out so a user can set them if they like.

Reply:
I commented them, the users can set them as they wish.